### ISSUE TITLE
GlomError.wrap(TypeError) - trigger_component_key is None in is_visible_in_frontend

### ISSUE THREAD

**Issue by SilviaAmAm** [open-formulieren/open-forms#2814]
### Product versie / Product version

2.0.0

### Omschrijf het probleem / Describe the bug

sentry DH open-forms 326223

Stacktrace

```
GlomError.wrap(TypeError): error raised while processing, details below.
 Target-spec trace (most recent last):
 - Target: {}
 - Spec: None
TypeError: expected spec to be dict, list, tuple, callable, string, or other Spec-like type, not: None
  File "django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "openforms/utils/decorators.py", line 23, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "openforms/submissions/api/viewsets.py", line 349, in summary
    summary_data = submission.render_summary_page()
  File "openforms/submissions/models/submission.py", line 431, in render_summary_page
    for node in renderer:
  File "openforms/submissions/rendering/renderer.py", line 106, in __iter__
    yield from child
  File "openforms/submissions/rendering/base.py", line 56, in __iter__
    yield from self.get_children()
  File "openforms/submissions/rendering/nodes.py", line 77, in get_children
    yield from FormioNode(step=self.step, renderer=self.renderer)
  File "openforms/submissions/rendering/base.py", line 56, in __iter__
    yield from self.get_children()
  File "openforms/formio/rendering/nodes.py", line 223, in get_children
    yield from child_node
  File "openforms/formio/rendering/nodes.py", line 151, in __iter__
    if not self.is_visible:
  File "openforms/formio/rendering/nodes.py", line 99, in is_visible
    if not is_visible_in_frontend(self.component, self.step.data):
  File "openforms/formio/utils.py", line 291, in is_visible_in_frontend
    trigger_component_value = glom(data, trigger_component_key, default=None)
  File "glom/core.py", line 2181, in glom
    raise err
```

### Stappen om te reproduceren / Steps to reproduce

Not investigated yet

### Screen resolution

None

### Device

None

### OS

None

**Comment by sergei-maertens** [comment]
Essentially a duplicate of #2385, was fixed in 2.0.2.

**Comment by sergei-maertens** [comment]
PR added with some more robustness tests, and we'll leave it at that :)
